mavros_msgs/msg/EstimatorStatus.msg

```diff
@@ -0,0 +1,17 @@
+# Current autopilot estimator state
+#
+# https://mavlink.io/en/messages/common.html#ESTIMATOR_STATUS_FLAGS
+
+std_msgs/Header header
+bool attitude_status_flag
```

Comment:
@saifullah3396 please split the fields by subject. Like
```
bool velocity_horiz_status_flag
bool velocity_vert_status_flag

bool pos_horiz_rel_status_flag
bool pos_horiz_abs_status_flag

bool pos_vert_abs_status_flag
bool pos_vert_agl_status_flag
```

Reply:
@kshahar can you address this so we can merge it? Thanks!

Reply:
Sorry, been on vacations! I'll do this tomorrow if it's not fixed until then.